src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -123,4 +125,14 @@ public static KeyToValueMappingEstimator MapKeyToValue(this TransformsCatalog.Co
             IComponentFactory<IMultiStreamSource, IDataLoader> loaderFactory = null)
             => new ValueToKeyMappingEstimator(CatalogUtils.GetEnvironment(catalog), columns, file, termsColumn, loaderFactory);
     }
+
+    public static class ToMappedValueCatalog
+    {
+        public static ValueMappingEstimator<TInputType, TOutputType> ValueMap<TInputType, TOutputType>(
```

Comment:
>ValueMap [](start = 69, length = 8)

I would call it MapKeyToValue, but we already have that function.
Maybe MapKeysToValues.

Honestly I don't like current name for function MapKeyToValue, it's misleading, maybe we should rename it to KeyToOriginalValue? 
@sfilipi, do you have any thoughts?

Reply:
Code has changed - this function no longer exists.

Reply:
I would totally call  it MapKeyToValue; and it makes more sense to do so. since the functionality provided is similar.

---
In reply to: [237660412](https://github.com/dotnet/machinelearning/pull/1710#discussion_r237660412) [](ancestors = 237660412)